pkgs/by-name/as/asm-lsp/package.nix

```diff
@@ -8,14 +8,14 @@ let
   pname = "asm-lsp";
   version = "0.4.2";
 in
-rustPlatform.buildRustPackage {
+rustPlatform.buildRustPackage rec {
   inherit pname version;
```

Comment:
Version needs to be updated to `asm-lsp-yyyy-mm-dd-unstable` if you insist on building from an untagged commit